libs/tx-builder-feature/src/utils/cache.ts

```diff
@@ -0,0 +1,106 @@
+import { ABI, ArbitraryState, ValidNetwork } from '@daohaus/common-utilities';
+import localforage from 'localforage';
+
+const defaultABIStore = {
+  '0x1': {},
+  '0x4': {},
+  '0x5': {},
+  '0x2a': {},
+  '0xa': {},
+  '0x64': {},
+  '0x89': {},
+  '0xa4b1': {},
+  '0xa4ec': {},
+};
+
+enum StoreName {
+  ABI = 'ABI',
```

Comment:
oh nice - so we can add another if needed.

Reply:
Yep. We'll need a few of these.